_sass/_person.sass

```diff
@@ -236,3 +237,14 @@
       color: $color-grey-dark
       text-transform: uppercase
 
+ a
```

Comment:
This will match all links on the site. Needs to be under .person-show-title. And given that the anchor is 'up-a-level' you can apply the color direct to that selector.